scripts/cypress/src/base.config.ts

```diff
@@ -56,6 +56,7 @@ const cypressWebpackConfig = (): Configuration => {
 
 type BaseConfig = Omit<Cypress.ConfigOptions, 'component'> & {
   component: Omit<Cypress.ConfigOptions['component'], 'devServer'> & {
+    specPattern?: string[];
```

Comment:
Add support for spec pattern to be passed. This is important to be able to specify tests via config.